tests/integration-tests/tests/update/test_update.py

```diff
@@ -148,7 +148,7 @@ def test_update_hit(region, scheduler, pcluster_config_reader, clusters_factory,
     bucket.upload_file(str(test_datadir / "postinstall.sh"), "scripts/postinstall.sh")
 
     # Create cluster with initial configuration
-    init_config_file = pcluster_config_reader()
+    init_config_file = pcluster_config_reader(resource_bucket=bucket_name)
```

Comment:
Just out of curiosity: why are we passing the resource bucket here? What are you testing here that we didn't in the other test?

Reply:
Updated PR description to explain test coverage